### DISCUSSION TITLE
With each singleboostr version some games is not showing

### DISCUSSION THREAD

**Discussion by jacquesPatat** [Ni1kko/HourBoostr#11] [Q&A]
i have noticed with 4.1.9, 4.2.1 and 4.2.5 some games is not appearring on the list, example i want to add game time to the original Alone in the Dark series but only picks up the new Alone in the dark games. Also with Grand theft auto, i want to add game time to GTA 3, vice city, san andreas, IV and V, but it only shows V with 4.2.5. i'm using the 4.2.1 to add time to IV and V.  And some games have two or more on the list (noticed the steam ID is the same on the list)

I also noticed something small, not sure if it got to do with what i mentioned, with version 4.1.9, 4.2.1 and 4.2.5 my game amount is in each one different.

**Comment by Ni1kko** [comment]
Thanks so much for taking the time to let us know, I have tested what you have said and i can see that some of my `Grand theft auto` titles are not showing in the list and like you said some other titles too. 

I wont be fixing on this application but i will make sure this is not an issue with the beta application. I will hopefully have beta ready for testing in next few days .

There will be new idler game library view in the beta where you can choose `Games/Tools/Software/DLC/Other` similar to the steam library view. i may also add a scheduler so you can have games idle only at certain times. Also in beta you will be able to switch easily between idling the account you are logged into on steam client whilst idling other account at the same time you can navigate from singleBoosr to multiBoostr at anytime

**Reply by jacquesPatat** [reply]
Awesome. will wait for the beta.

**Comment by mysza224** [comment]
if you still got the problem check out new post on issues channel

**Reply by jacquesPatat** [reply]
thanks. will try later out
